console/atest-ui/src/App.vue

```diff
@@ -99,6 +118,20 @@ const handleSelect = (key: string) => {
     </el-aside>
 
     <el-main style="padding-top: 5px; padding-bottom: 5px;">
+      <div style="position: absolute; top: 10px; right: 20px;">
+        <el-col style="display: flex; align-items: center;">
+          <el-tag style="font-size: 18px;">{{ t('lan') }}</el-tag>
+          <el-dropdown trigger="click" @command="(command: string) => handleChangeLan(command)">
+            <el-icon><arrow-down /></el-icon>
+            <template #dropdown>
+              <el-dropdown-menu>
+                <el-dropdown-item command="chinese">中文</el-dropdown-item>
```

Comment:
Is that possible to add a new option `auto`? Let it read the locale setting from the user's browser.

Reply:
What read the locale setting from the user's browser means? I have set up to record the selection to localStorage.

Reply:
Users can set the language in their browser. For example

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/b323f431-3e41-43e2-a9e8-a5a7087434d4)

I think you don't give it a specify value then it will read the locale from browser.